runtime/src/bpf_helper.cpp

```diff
@@ -727,7 +758,7 @@ const bpftime_helper_group shm_maps_group = { {
 } };
 
 extern const bpftime_helper_group extesion_group;
-
+#if __linux
```

Comment:
`__linux` or `__linux__`? If both of them are available, only choose one option

Reply:
should be `__linux__` it's a typo. Thanks for pointing out.